lib/Notification/Notifier.php

```diff
@@ -79,7 +79,7 @@ public function prepare(INotification $notification, $languageCode) {
 				}
 
 				$notification->setParsedSubject($subject)
-					->setLink($this->url->getAbsoluteURL('index.php/settings/personal'));
+					->setLink($this->url->getAbsoluteURL('index.php/settings/user'));
```

Comment:
Should use `getAbsoluteRouteURL()` in first place

Reply:
@nickvergessen i guess you refer to ``linkToRouteAbsolute()``?  

@jancborchardt then it should be ``$this-url->linkToRouteAbsolute('settings.PersonalSettings.index')``